sechub-pds-examples/gosec/docker/Dockerfile

```diff
@@ -0,0 +1,94 @@
+# SPDX-License-Identifier: MIT
+
+# The image argument needs to be placed on top
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+# The remaining arguments need to be placed after the `FROM`
+# See: https://ryandaniels.ca/blog/docker-dockerfile-arg-from-arg-trouble/
+ARG WORKSPACE="/workspace"
+ARG PDS_FOLDER="/pds"
+ARG TOOL_FOLDER="/tool"
+ARG PDS_VERSION=0.21.0
+ARG GOSEC_VERSION=2.8.0
+ARG GO=go1.16.5.linux-amd64.tar.gz
+
+# non-root user
+# using fixed group and user ids
+# gosec needs a home directory for the cache
+RUN groupadd --gid 2323 gosec \
+     && useradd --uid 2323 --no-log-init --create-home --gid gosec gosec
+
+# Update image and install dependencies
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update && \
+    apt-get dist-upgrade --assume-yes && \
+    apt-get install --assume-yes wget openjdk-11-jre-headless && \
+    apt-get clean
+
+# Create tool folder
+RUN mkdir -p $TOOL_FOLDER
+COPY gosec.sh $TOOL_FOLDER/gosec.sh
+RUN chmod +x $TOOL_FOLDER/gosec.sh
+
+# Install Go
+RUN cd /tmp && \
+    # create checksum file
+    echo "b12c23023b68de22f74c0524f10b753e7b08b1504cb7e417eccebdd3fae49061 $GO" > $GO.sha256sum && \
```

Comment:
Maybe you could declare this sha256sum also in an argument at the top of this Dockerfile, so it can be easily changed together with the `$GO` ARG.

Reply:
done